### PR TITLE
Update version in pNFT guide

### PR DESCRIPTION
**Pull request by danenbm**
Also add Auth Rules, maybe its not needed in that guide.  If so I can remove auth rules from the guide.

### CODE REVIEW

File: token-metadata/program/ProgrammableNFTGuide.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 
-Developer packages:
-* :crab: Rust crate: [v1.8.0](https://crates.io/crates/mpl-token-metadata/1.8.0)
-* :package: NPM package: [v2.8.0](https://www.npmjs.com/package/@metaplex-foundation/mpl-token-metadata/v/2.8.0)
+## Developer packages
+Token metadata
+* :crab: Rust crate: [v1.8.1](https://crates.io/crates/mpl-token-metadata/1.8.1)
+* :package: NPM package: [v2.8.1](https://www.npmjs.com/package/@metaplex-foundation/mpl-token-metadata/v/2.8.1)
+
+Auth rules
```
Comment (febo) [inline token-metadata/program/ProgrammableNFTGuide.md:8]: Should we use the "full name"? E.g., Token Authorization Rules.

File: token-metadata/program/ProgrammableNFTGuide.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
 # Programmable NFT Guide
 
-Developer packages:
-* :crab: Rust crate: [v1.8.0](https://crates.io/crates/mpl-token-metadata/1.8.0)
-* :package: NPM package: [v2.8.0](https://www.npmjs.com/package/@metaplex-foundation/mpl-token-metadata/v/2.8.0)
+## Developer packages
+Token metadata
```
Comment (febo) [inline token-metadata/program/ProgrammableNFTGuide.md:4]: Nitpick: Token Metadata

File: token-metadata/program/ProgrammableNFTGuide.md (hunk 1 above)
Comment (danenbm) [reply token-metadata/program/ProgrammableNFTGuide.md]: Done

File: token-metadata/program/ProgrammableNFTGuide.md (hunk 2 above)
Comment (danenbm) [reply token-metadata/program/ProgrammableNFTGuide.md]: Done

**Review by febo — APPROVED** [review]
This PR updates the links to the developer packages, including Token Authorization Rules program.

**Review by blockiosaurus — APPROVED** [review]

**Pull request merged**